src/models/time.rs

```diff
@@ -198,8 +189,8 @@ impl UTCTime {
         let mut buf = Vec::with_capacity(13);
         buf.push((self.datetime.year() / 10 % 10) as u8 + b'0');
         buf.push((self.datetime.year() % 10) as u8 + b'0');
-        buf.push((self.datetime.month() / 10 % 10) as u8 + b'0');
-        buf.push((self.datetime.month() % 10) as u8 + b'0');
+        buf.push((self.datetime.month() as u8 / 10 % 10) + b'0');
```

Comment:
Why this change? It makes the code less consistent.

Reply:
This is because [`time::OffsetDateTime::month`](https://docs.rs/time/0.3.4/time/struct.OffsetDateTime.html#method.month) returns a [`Month`](https://docs.rs/time/0.3.4/time/enum.Month.html) enum variant, so this has to be cast before arithmetic can be performed.

Perhaps, since this one has to change, the rest could be changed to restore consistency.

Reply:
Oh then it's okay I guess.